src/styles/_calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  margin: auto;
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  width: $calendarWidth;
+  gap: 1px;
+  padding-top: 50px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 30px;
+    font-family: Arial, sans-serif;
```

Comment:
move it to the `hmtl` selector into normalize a file

Reply:
got it